nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/QueryCassandra.java

```diff
@@ -75,19 +81,64 @@
 @Tags({"cassandra", "cql", "select"})
 @EventDriven
 @InputRequirement(InputRequirement.Requirement.INPUT_ALLOWED)
-@CapabilityDescription("Execute provided Cassandra Query Language (CQL) select query on a Cassandra 1.x, 2.x, or 3.0.x cluster. Query result "
-        + "may be converted to Avro or JSON format. Streaming is used so arbitrarily large result sets are supported. This processor can be "
+@CapabilityDescription("Executes provided Cassandra Query Language (CQL) select query on a Cassandra to fetch all rows whose values"
```

Comment:
Thank you for your wording! I really appreciate it.

I think it is a little different from GenerateTableFetch. Because GenerateTableFetch use various kinds of DatabaseAdapter for RDB. They usually use "ORDER BY" keyword.

But Cassandra does not have this feature. And It has specialized structure based on Column Family, not Row

So I built this processor with range query.